docroot/themes/contrib/civictheme/civictheme_library/components/03-organisms/callout/callout.twig

```diff
@@ -66,6 +66,7 @@
                     text: link.text,
                     url: link.url,
                     icon: link.text ? '' : 'arrows_rightarrow_3',
+                    is_external: link.is_external,
```

Comment:
We have not been consistent on `is_external` - I think callout doesn't change icon with this variable but wanted to add it.